tony-cli/src/main/java/com/linkedin/tony/cli/NotebookSubmitter.java

```diff
@@ -82,17 +90,12 @@ public static void main(String[] args) throws  Exception {
     updatedArgs[args.length + 2] = "--conf";
     updatedArgs[args.length + 3] = TonyConfigurationKeys.APPLICATION_TIMEOUT + "=" + String.valueOf(24 * 60 * 60 * 1000);
 
-    TonyClient client = TonyClient.createClientInstance(updatedArgs, new Configuration());
+    client.init(updatedArgs);
+    client.start();
     if (client == null) {
       System.exit(-1);
     }
-    Thread clientThread = new Thread(() -> {
-      try {
-        client.run();
-      } catch (IOException | URISyntaxException | YarnException | InterruptedException e) {
-        LOG.error(e);
-      }
-    });
+    Thread clientThread = new Thread(client::start);
```

Comment:
Is `client.start()` being called twice? Once here and once on line 94? I think it only needs to be called here (triggered when the `Thread` is started)?